server/neptune/temporalworker/server.go

```diff
@@ -38,34 +35,23 @@ const (
 	ProjectJobsViewRouteName = "project-jobs-detail"
 )
 
-// Config is TemporalWorker specific user config
-type Config struct {
-	AtlantisURL     *url.URL
-	AtlantisVersion string
-	CtxLogger       logging.Logger
-	SslCertFile     string
-	SslKeyFile      string
-	Scope           tally.Scope
-	Closer          io.Closer
-	TemporalCfg     valid.Temporal
-	Port            int
-	App             githubapp.Config
-}
-
 type Server struct {
 	Logger          logging.Logger
 	HttpServerProxy *neptune_http.ServerProxy
 	Port            int
 	StatsScope      tally.Scope
 	StatsCloser     io.Closer
 	TemporalClient  client.Client
-	Activities      *workflows.Activities
+
+	DeployActivities    *workflows.DeployActivities
+	TerraformActivities *workflows.TerraformActivities
+	GithubActivities    *workflows.GithubActivities
```

Comment:
? I don't think this needs to be separate, my comment was about keeping each workflow's activities separate

Reply:
As discussed during standup, I separated out github activities to avoid registering the activity twice since both the deploy and terraform workflow uses it.